v2/emailpassword/common-customizations/signup-form/adding-fields.mdx

```diff
@@ -58,9 +248,13 @@ SuperTokens.init({
 });
 ```
 </TabItem>
-</FrontendSDKTabs>
+</FrontendPreBuiltUITabs>
+
+<img alt="Prebuilt form UI with extra custom fields" src="/img/^{codeImportRecipeName}/signup-with-name-and-age.png" />
```

Comment:
this image is too large.. you can set the style tag of this html to make it smaller.

Reply:
IMO image for emailpassword is consistent with the rest of images,
I've updated the image for thirdpartyemailpassword which was appearing very huge

Reply:
unless you want the height of the images to be reduced, lmk